Sources/Plasma/FeatureLib/pfPython/plPythonCallable.h

```diff
@@ -54,102 +55,90 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "plProfile.h"
 
 #include "cyPythonInterface.h"
+#include "plPythonConvert.h"
 #include "pyGlueHelpers.h"
 #include "pyObjectRef.h"
 
 plProfile_Extern(PythonUpdate);
 
-namespace plPythonCallable
+namespace plPython
```

Comment:
Why was this namespace changed? The file name is still `plPythonCallable.h`. Is it OK for the filename and namespace/class in the file not to match?

Reply:
Yeah, there's no requirement for namespaces to match filenames (or have a name at all, which is another barrel of monkeys). I changed the name because, previously, all of the machinery I had written was specifically for calling Python callables. Now, there is a batch of general object conversion functions in plPythonConvert.h, so I wanted the namespace name to be more general.